spec/formalism/form/coercion_spec.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+## This is unit test, created to simplify testing of different coercions
+## You can find integration test in spec/formalism/form/fields_spec.rb
+## (Formalism::Form -> .field -> coersion)
+##
+## In tests `eql` used instead of `eq` to determine result type
+describe 'Formalism::Coercion' do
+	let(:value) {}
+
+	let(:field_defenition) { [type] }
+	let(:form) do
+		## Pass method call result throug instance_eval inside Class.new
+		field_defenition = self.field_defenition
+
+		Class.new(Formalism::Form) do
+			field :value, *field_defenition
+			alias_method :execute, :value
```

Comment:
Why?

Reply:
I used to use `alias_method` instead of 
```ruby 
def execute
  value
end
```

It looks same for me, but i can expand if you want to

Reply:
I guess, you can write:

```ruby
form.new(value: value).value
```

And you don't need `run` at all.

Reply:
> undefined local variable or method `execute'

"return value" seems better than "do nothing"

Reply:
> undefined local variable or method `execute'

Don't call `run`, re-check the code (from me).

Reply:
Fixed